language/functional_tests/_old_move_ir_tests/src/tests/epilogue.rs

```diff
@@ -296,178 +87,6 @@ fn gas_charge_accurate() {
     ));
 }
 
-#[test]
-fn gas_deposit_withdraws() {
```

Comment:
I deleted this for two reasons:
1) It's not buying us much.
2) No easy way for us to test that the amount of gas in the write set is really really the same as the difference of balances before and after.